Prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+Emails Task
+Mischa Burgess
+"""
+
+
+def main():
+    # global name, email
+    name_from_email = {}  # create empty dictionary
+    email = input("Email: ")
+    while email != "":  # while email is not empty
+        name = email_to_name_function(email)  # get name from email using function
+        response = input("Is your name {}? (y/n) ".format(name))  # check name is correct
+        if response.lower() != "y" and response != "":  # is response is 'n'
```

Comment:
Consider the user may possibly mean to press y, but misstyped.

Reply:
Good suggestion, thank you